### PR TITLE
a try of docs-refactoring

### PR DESCRIPTION
**Pull request by plyfager**
1. Basiclly follow the given structure.
2. It seems lack some documents, althogh I split some mds into new ones, but it seems not enugh.

### CODE REVIEW

File: docs/modelzoo_statistics.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+
+# Model Zoo
```
Comment (congee524) [inline docs/modelzoo_statistics.md:2]: The file should not be traced.
`modelzoo_statistics.md` will be generated by `stat.py` which is called in `conf.py`.

**Comment by congee524** [comment]
you should modify the file `stat.py` to fix the problem that there are two `Model Zoo` in the content like this 
image

The title in the content is decided by the first-level title in the markdown file. Since the first-level title of `modelzoo_statistics.md` is `Model Zoo`, the title in the content is also `Model Zoo`

**Comment by nbei** [comment]
@plyfager , you may fix the double `module_zoo` problem according to the guidance from @congee524 .

File: docs/modelzoo_statistics.md (hunk 1 above)
Comment (congee524) [reply docs/modelzoo_statistics.md]: The file still seems to be tracked.

**Pull request merged**